semver/types.ts

```diff
@@ -28,7 +28,7 @@ export type Operator = typeof OPERATORS[number];
 export interface Comparator extends SemVer {
   operator: Operator;
   /**
-   * @deprecated (will be removed in 0.216.0) {@linkcode Comparator} extends {@linkcode SemVer}. Use `major`, `minor`, `patch`, `prerelease`, and `build` properties instead.
+   * @deprecated (will be removed after 0.216.0) {@linkcode Comparator} extends {@linkcode SemVer}. Use `major`, `minor`, `patch`, `prerelease`, and `build` properties instead.
```

Comment:
```suggestion
   * @deprecated (will be removed after 0.217.0) {@linkcode Comparator} extends {@linkcode SemVer}. Use `major`, `minor`, `patch`, `prerelease`, and `build` properties instead.
```
To align with the other APIs.

Reply:
Oops! Thanks